app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] in order to make this work you should set the foreign key to 'author_id' for the comments and the likes as well